docs/contributing.md

```diff
@@ -12,13 +12,24 @@ npm start
 
 All packages will be live-built, and a live development server showcasing components with hot reload enabled will then run at [localhost:8080](http://localhost:8080).
 
+### First time step
+
+If this is the first time you have cloned the repo, run the `npm run prepare` script that will set up `husky` to provide a git precommit hook that will format and lint any code you have added to a PR.
```

Comment:
`prepare` should automatically run after `install` but I guess it doesn't hurt to have this

Reply:
It doesn't seem to in the lerna world 🤷🏼